duckduckgo/iOS
# Restore Subscriptions

Task/Issue URL: https://app.asana.com/0/72649045549333/1205054784245717/f

**Description**:
Implements subscription restoration via Apple ID

:warning: Test on device > Alpha debug scheme

**Steps to test this PR**:
1.  Purchase a subscription
2. Go to Settings > Privacy Pro > Subscription Settings
3. Tap "Remove From This Device" and Confirm
4. In Settings > Privacy Pro, tap "Learn More"
5. Tap "Restore Purchase"
6. Tap "Restore" in the Apple ID Section
7. Confirm the subscription is restored and you can manage it again
8. Go back to Step 2
9. Try to purchase a subscription again (not restore)
10. Confirm your existing subscription is restored.
